typed_sql/test/typed_sql/testrunner.dart

```diff
@@ -151,3 +153,13 @@ final class TestRunner<T extends Schema> {
     }
   }
 }
+
+extension JsonValueSubjectExt on Subject<JsonValue> {
```

Comment:
How about a moving this to a `checks.dart` or `checks_helper.dart` file and re-exporting it from `testrunner.dart`?